cram_common/cram_common_failures/package.xml

```diff
@@ -17,7 +17,13 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>cram_designators</build_depend>
   <build_depend>cram_language</build_depend>
+  <build_depend>cram_tf</build_depend>
+  <build_depend>roslisp</build_depend>
 
+  <run_depend>cram_designators</run_depend>
   <run_depend>cram_language</run_depend>
+  <run_depend>cram_tf</run_depend>
+  <run_depend>roslisp</run_depend>
```

Comment:
Now that you're at it anyway, could you please change the package.xml to version v2?
Such that we don't have any ``build_depend`` and ``run_depends`` but just ``depend``?
See, for example, https://github.com/cram2/cram/blob/master/cram_common/cram_object_interfaces/package.xml